easybuild/easyconfigs/g/GObject-Introspection/GObject-Introspection-1.60.0-GCCcore-8.2.0-Python-3.7.2.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'ConfigureMake'
+
+name = 'GObject-Introspection'
+version = '1.60.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://wiki.gnome.org/GObjectIntrospection/'
+description = """GObject introspection is a middleware layer between C libraries
+ (using GObject) and language bindings. The C library can be scanned at
+ compile time and generate a metadata file, in addition to the actual
+ native C library. Then at runtime, language bindings can read this
+ metadata and automatically provide bindings to call into the C library."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['9efe4090cb59717126701e97062e784773f800b8d47af14c4d278ebf194df35d']
+
+dependencies = [
+    ('Python', '3.7.2'),
```

Comment:
Should this use multi-deps?

Reply:
We can't. It's not really a python package, so it doesn't really make sense to start with, but, just to hammer in the last nail, I just tried with 1.60.1 and
```
checking whether python2.7 version is >= 3.4... no
configure: error: Python interpreter is too old
```